include/asm/disassembler.h

```diff
@@ -20,31 +20,39 @@
 # define CONNOT_OPEN_FILE_ERR "Cannot open file!"
 # define UNEXPECTED_EOF_ERR "Unexpected end of file!"
 # define WRONG_FILE_FORMAT_ERR "Wrong file format!"
-# define SIZE_MISMATCH_ERR "Size mismatch!"
+# define INVALID_PAR_TYPE_ERR "Invalid parameter type!"
+# define INVALID_CMD_TYPE_ERR "Invalid command type!"
 
 typedef struct		s_headdata
 {
-	short			filler2;
 	char			champ_comment[COMMENT_LENGTH];
 	uint32_t		size;
```

Comment:
it's better to place uint32_t at first place

Reply:
Does it change anythin? 2, 1, 4 or 4, 2, 1 - looks the same for me.